### ISSUE TITLE
NTR: CT or MRI: brainstem lucencies (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8889]
New term request CT or MRI: brainstem lucencies (segal_071120214429): Midbrain; pons; medulla; pontocerebellar

**Comment by pnrobinson** [comment]
@MickeySegal
these are probably different findings for CT and MR? Please provide more context

**Comment by MickeySegal** [comment]
We are changing such findings to the form "Imaging: brainstem lucencies"

Is that what you are asking?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000916
- label: Brainstem lucency
- parents: [Abnormal brainstem morphology (HP:0002363)](https://hpo.jax.org/app/browse/term/HP:0002363)
- pmid: 
- def: A lesion located in the brainstem that appears areas less dense on magnetic resonance imaging or lighter on computed tomography.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8889
- synonyms:
